Documentation/network/bgp-control-plane-troubleshooting.rst

```diff
@@ -0,0 +1,72 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _bgp_control_plane_troubeshooting:
+
+BGP Control Plane Troubleshooting Guide
+=======================================
+
+This document enumerates typical troubles and solutions by configuring the BGP
+Control Plane.
+
+**Even though CiliumBGPPeeringPolicy was applied, the BGP peering is not established**
```

Comment:
Would you mind changing these bold statements into sub-headers? I think that would be more appropriate given the context.

Reply:
Makes sense. Fixed!